### DISCUSSION TITLE
十月更新 - 2

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#1536] [通知]
## 🐛 Bug 修复

- 修复在进入直播时没有应用默认播放模式的问题 @Richasy (#1535)
- 修复部分PGC额外内容因标题重复导致解析失败的问题 @Richasy (#1534)
- 修复综合动态序列解析溢出的问题 @Richasy (#1533)
- 修复播放器显示模式文本空白的问题 @Richasy (#1532)
- 修复关注的直播列表滚动导致的错位问题 @Richasy (#1522)

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2210.2.0'>十月更新 - 2</a>.</em>

**Comment by SoDebug** [comment]
好像没办法评论
